src/plugins/rovpilot/public/js/rovpilot.js

```diff
@@ -379,11 +379,7 @@
       //to deal with the clocks not being in sync between the computer and the ROV.
       var self = this;
 
-      //Initial sync of state
-      self.cockpit.rov.emit('plugin.rovpilot.getState', function(state) {
```

Comment:
So, the powerLevel is not a Global setting with the ROV but instead just a client setting. So rovpilot.js is actually the owner of the state (thus why it sets the default value).  No need to try and sync state with the ROV